### PR TITLE
Move dependencies to dev dependencies

### PR DESCRIPTION
**Pull request by fourlastor**
There were a bunch of dev dependencies incorrectly added as dependencies, this addresses it.

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/ca28365efd156360a63f97994c8130531935b68b/index.html

File: packages/article/package.json [hunk 1]
```diff
@@ -56,7 +57,6 @@
     "styled-components": "3.2.2"
   },
   "dependencies": {
-    "@storybook/addon-knobs": "3.3.15",
```
Comment (colincclark) [inline packages/article/package.json]: this is used here: https://github.com/newsuk/times-components/blob/master/packages/article/article.stories.js#L8

Comment (fourlastor) [reply packages/article/package.json]: Moved to dev dependencies

File: packages/tealium/package.json [hunk 2]
```diff
@@ -21,11 +21,8 @@
     "url": "https://github.com/newsuk/times-components/issues"
   },
   "homepage": "https://github.com/newsuk/times-components#readme",
-  "dependencies": {
-    "@storybook/addon-actions": "3.3.15",
-    "@storybook/addons": "3.3.15"
```
Comment (killanaca) [inline packages/tealium/package.json]: add as dep

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/8defa2d5489f023fe7228d263ce370f3cb99b0bb/index.html

**Pull request merged**